resources/phing/AtoumTask.php

```diff
@@ -171,6 +178,21 @@ public function execute()
 			}
 		}
 
+		if ($this->telemetryEnabled())
+		{
+			if (class_exists('\mageekguy\atoum\reports\telemetry') === false) {
+				throw new exception('AtoumTask depends on atoum/reports-extension being installed to enable telemetry report');
+			}
+
+			$telemetry = new \mageekguy\atoum\reports\telemetry();
```

Comment:
@oxman could you use the short name for this class: `reports\telemetry` ? We already have the required `use` statement.